src/theme.jsx

```diff
@@ -18,6 +35,24 @@ const theme = createMuiTheme({
     caption: Roboto400,
     button: Roboto500,
   },
+  overrides: {
+    MuiTypography: {
+      display2: {
+        color: AQUA,
+      },
+    },
+    MuiTableCell: {
+      head: {
+        fontSize: 16,
+        color: '#64A4C9',
+      },
+    },
+    MuiCircularProgress: {
+      colorPrimary: {
+        color: AQUA,
+      },
+    },
+  },
 });
```

Comment:
From https://material.io/design/color/the-color-system.html#color-theme-creation, a primary color is the color displayed most frequently. The secondary color is used to serve as visual contrast to the primary color and is mainly used for things like action buttons, swtiches, links, etc.

To that end, we need to tweak the theme a little. In our case, the primary color of the app is blue and the secondary color is the aqua color. I used https://material.io/tools/color/#!/?view.left=0&view.right=0&primary.color=01579B&secondary.color=80DEEA to help generate the colors for primary and secondary palette.

```js
primary: {
  light: '#4f83cc',
  main: '#01579b',
  dark: '#002f6c',
  contrastText: '#fff',
},
secondary: {
  light: '#b4ffff',
  main: '#80deea',
  dark: '#4bacb8',
  contrastText: '#000',
},
```

Next, the default background color we will use is the light grey color. Using the color.io link above, we can grab the color via the dev-tools color extraction utility.

```js
const LIGHT_GREY = '#F5F5F6';
```

The pallette now becomes:

```js
palette: {
  background: {
    default: LIGHT_GREY,
  },
  type: 'light',
  primary: {
    light: '#4f83cc',
    main: '#01579b',
    dark: '#002f6c',
    contrastText: '#fff',
  },
  secondary: {
    light: '#b4ffff',
    main: '#80deea',
    dark: '#4bacb8',
    contrastText: '#000',
  },
},
```

Now that our palette has been changed, we need to tweak our views/components to look just like before. I'll help for the project list view and you can take care of the project view.